shard/src/guild_settings.py

```diff
@@ -91,6 +91,15 @@ def starboard_threshold(self, new_threshold: int):
         self._settings_dict['starboard_threshold'] = new_threshold
         self._update_db()
 
+    @property
+    def gulag_emoji(self) -> str:
+        return self._settings_dict['gulag_emoji'] if 'gulag_emoji' in self._settings_dict else "⚒️"
```

Comment:
this means that current servers will have to change their settings to use their custom gulag emoji's again

Reply:
pretty sure about 3 server use it, so better now than later